jdi-light-html-tests/src/test/java/io/github/epam/example/common/ColorPickerExample.java

```diff
@@ -0,0 +1,57 @@
+package io.github.epam.example.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.colorPicker;
+import static io.github.com.pages.HtmlElementsPage.disabledPicker;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+
+public class ColorPickerExample implements TestsInit {
+    final private static String COLOR = "#ffd7a6";
```

Comment:
I reckon the value of the colour is wrong here since checkColorTest fails
```suggestion
    final private static String COLOR = "#3fd7a6";
```